### DISCUSSION TITLE
Add customizable card styles

### DISCUSSION THREAD

**Discussion by biglonglong** [chrede88/qubt#184] [Ideas]
First of all, I really like the theme, but I still think it's a bit too simple, I have an idea here: we can add a card style for displaying whatever we want on any page. Maybe I didn't describe it clearly enough, but here's an equally nice site for your reference that might give you more inspiration for your Qubt theme design.The URL is https://skywt.cn/
Good Luck!

**Comment by chrede88** [comment]
You are looking for a variable card size for the blog part? i.e. [here](https://chrede88.github.io/qubt/blog/)
Or do you also want to add cards on the main page?

**Reply by biglonglong** [reply]
Yes, maybe we can add some cards to the Home page/About page to display some personalized content, like https://skywt.cn/lab/.
I'm not good at front-end technology, but I've seen similar techniques on other themes' Friendlink page, like https://skywt.cn/friends/.
Thank you very much for your prompt reply.

**Reply by chrede88** [reply]
I'll consider this. My original goal with the site was to keep it minimalistic. I'll play around with some different styles and ways of implementing this and return when I've made up my mind about this idea.

**Comment by biglonglong** [comment]
OK，thank you！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Christian ***@***.***&gt;; 
发送时间: 2024年11月27日(星期三) 凌晨1:30
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [chrede88/qubt] Add customizable card styles (Discussion #184)














 
I'll consider this. My original goal with the site was to keep it minimalistic. Play around with some different styles ways of implementing this and return when I've made up my mind about this idea.
